JenkinsConsoleUtility/jcuSrc/Commands/CloudScriptListener.cs

```diff
@@ -128,20 +128,15 @@ private int FetchTestResult(string workspacePath, TestTitleData testTitleData)
             var callResult = ExecuteCloudScript(CsFuncGetTestData, _getRequest, testTitleData.extraHeaders, out testResults, out errorReport);
 
             string outputFileFullPath = null;
-            for (int i = 0; i < 100; i++)
-            {
-                string eachOutputFile = Path.Combine(workspacePath, "ListenCsResult" + i + ".xml");
-                if (!File.Exists(eachOutputFile))
-                {
-                    outputFileFullPath = eachOutputFile;
-                    break; // Find the first file that doesn't exist
-                }
-            }
+
+            string outputFileName = "ListenCsResult.xml";
+            
+            outputFileFullPath = Path.Combine(workspacePath, outputFileName);
 
             if (!callResult || testResults == null)
                 return 1;
             JcuUtil.FancyWriteToConsole(ConsoleColor.Gray, "Writing test results: " + outputFileFullPath);
-            return JUnitXml.WriteXmlFile(outputFileFullPath, testResults, true);
+            return JUnitXml.WriteXmlFile(outputFileFullPath, testResults, false);
```

Comment:
This will cause for the new results to be appended at the end of the file. Is this desired behavior?